src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1690,15 +1742,19 @@
 
 helps['network vnet check-ip-address'] = """
     type: command
-    short-summary: Check if a private IP address is available for use.
+    short-summary: Check if a private IP address is available for use within a virtual network.
+    examples:
```

Comment:
Remove or improve. A possible improvement would be to query an existing VNET subnet to get its address range, and then check all available IP addresses within that subnet to see what is available for static assignment.